.gitignore

```diff
@@ -1,89 +1,92 @@
-# Byte-compiled / optimized / DLL files
-__pycache__/
-*.py[cod]
-*$py.class
-
-# C extensions
-*.so
-
-# Distribution / packaging
-.Python
-env/
-build/
-develop-eggs/
-dist/
-downloads/
-eggs/
-.eggs/
-lib/
-lib64/
-parts/
-sdist/
-var/
-*.egg-info/
-.installed.cfg
-*.egg
-
-# PyInstaller
-#  Usually these files are written by a python script from a template
-#  before PyInstaller builds the exe, so as to inject date/other infos into it.
-*.manifest
-*.spec
-
-# Installer logs
-pip-log.txt
-pip-delete-this-directory.txt
-
-# Unit test / coverage reports
-htmlcov/
-.tox/
-.coverage
-.coverage.*
-.cache
-nosetests.xml
-coverage.xml
-*,cover
-.hypothesis/
+.DS_Store
 
-# Translations
-*.mo
-*.pot
+# Created by .ignore support plugin (hsz.mobi)
+### Java template
+*.class
 
-# Django stuff:
-*.log
-local_settings.py
+# Mobile Tools for Java (J2ME)
+.mtj.tmp/
 
-# Flask stuff:
-instance/
-.webassets-cache
+# Package Files #
+*.jar
+*.war
+*.ear
 
-# Scrapy stuff:
-.scrapy
+# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
+hs_err_pid*
+### JetBrains template
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio
 
-# Sphinx documentation
-docs/_build/
+*.iml
 
-# PyBuilder
-target/
+## Directory-based project format:
+.idea/
```

Comment:
I believe this should cover all of the contents of `.idea/` below

Reply:
I'd rather keep this as close as possible to the original. I think most of this stuff does not apply to this project. You can move it to a `.gitignore_global` if you need it for your own dev environment.

Reply:
fixed in next commit